tools/mcctl-guides/mcctl-reference/metric-and-usage-commands/index.html

```diff
@@ -10,13 +10,13 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Metric and Usage Commands | MobiledgeX Developers</title>
+  <title>Metric and Usage Commands | EdgeXR Developers</title>
 <meta name="description" content="Learn about metric and usage commands with mcctl utility" />
 <meta property="og:type" content="website" />
 <meta property="og:title" content="Metric and Usage Commands" />
 <meta property="og:description" content="Learn about metric and usage commands with mcctl utility" />
 <meta property="og:url" content="http://dev-v3.test/tools/mcctl-guides/mcctl-reference/metric-and-usage-commands" />
-<meta property="og:site_name" content="MobiledgeX Developers" />
+<meta property="og:site_name" content="EdgeXR Developers" />
```

Comment:
Just a comment, but a lot of the data in the head is the same is in each file. I wonder if there's some way to define a common header. I also wonder if these files were generated from some source docs that we don't have anymore.

Reply:
As mentioned above, it's all generated, so the best way to handle these would be to fix the origin and regen the static page using different base pages